cartographer/mapping/proto/trajectory.proto

```diff
@@ -48,4 +49,7 @@ message Trajectory {
 
   // Submaps associated with the trajectory.
   repeated Submap submap = 2;
+
+  // Time-ordered IMU data.
+  repeated sensor.proto.ImuData imu_data = 4;
```

Comment:
The `Trajectory` message is contained in the `SparsePoseGraph` proto of which we have a single one at the beginning of each `.pbstream`. Putting a lot of data into a single proto message is not a good idea, so we should not put this here. Put in `SerializedData`?